py/BUILD.bazel

```diff
@@ -467,7 +468,8 @@ BIDI_TESTS = glob(["test/selenium/webdriver/common/**/*bidi*_tests.py"])
                 "test/selenium/webdriver/common/**/*.py",
                 "test/selenium/webdriver/support/**/*.py",
             ],
-            exclude = ["test/selenium/webdriver/common/print_pdf_tests.py"],
+            exclude = ["test/selenium/webdriver/common/print_pdf_tests.py"] +
```

Comment:
It might be easier to add these test targets to `.skipped-tests`.

Reply:
I am not even sure why `print_pdf_tests.py` is being excluded, it works fine on all browsers. I will enable it in a separate PR and see if CI has any issues with it.